providers/dns/digitalocean/digitalocean.go

```diff
@@ -6,6 +6,8 @@ import (
 	"fmt"
 	"net/http"
 	"sync"
+
+	"github.com/xenolf/lego/acme"
 )
 
 // DNSProviderDigitalOcean is an implementation of the DNSProvider interface
```

Comment:
This should be corrected to "an implementation of the `acme.ChallengeProvider` interface" (in the other DNS provider packages too.)